src/containers/Header.js

```diff
@@ -1,9 +1,6 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import { connect } from 'react-redux';
-import { bindActionCreators } from 'redux';
```

Comment:
Technically, these changes don't belong to this PR and hence would be best pushed to `master` directly..